Jenkinsfile

```diff
@@ -16,48 +16,64 @@ pipeline {
     stage('Linting') {
       parallel {
         stage('Code') {
-          steps { sh './bin/test_linting' }
+          steps { echo './bin/test_linting' }
         }
 
         stage('Changelog') {
-          steps { sh './bin/test_changelog' }
+          steps { echo './bin/test_changelog' }
```

Comment:
curious, why this change?

Reply:
temporary to speed up the build

Reply:
Are you done with the implementation? If so and only minor tweaks remain, I would switch these back

Reply:
@eranha